packages/components/pie-checkbox-group/src/checkbox-group.scss

```diff
@@ -1 +1,53 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-checkbox-group {
+    --checkbox-group-offset: var(--dt-spacing-c);
+    --checkbox-group-offset--inline: var(--dt-spacing-e);
+    --checkbox-group-label-offset: var(--dt-spacing-a);
+
+    margin: 0;
+    padding: 0;
+    border: 0;
+    min-width: 0;
+
+    ::slotted(pie-checkbox) {
+        display: flex;
+    }
+
+    &:not(.__inline) {
+        ::slotted(pie-checkbox:not(:first-child)) {
+            margin-block-start: var(--checkbox-group-offset);
+        }
+    }
+}
+
+.__inline {
+    display: flex;
+
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.label {
+    max-width: 100%;
+    white-space: normal;
+    padding: 0;
+    font-size: p.font-size(--dt-font-body-strong-s-size);
+    line-height: p.line-height(--dt-font-body-strong-s-line-height);
+    font-weight: var(--dt-font-body-strong-s-weight);
+    color: var(--dt-color-content-default);
+
+    margin-bottom: var(--checkbox-group-label-offset);
```

Comment:
Looks like we've got a mix of logical and non-logical properties, do you think we should align them?

Reply:
Sure, great catch, I will update now 👍